<launch>
    <param name="serverNameOrIP" value="10.7.7.99"/> 
    
    <node pkg="tf" type="static_transform_publisher" name="kinect_base_world" args="1.1573  1.2308 0.9893  -0.7099  0.0415   2.1197 /world /head/skel_depth_frame 100"/>
    
<!--K1- old Human tracking trial 1 : 1.0327  1.4245 1.1351   2.5764 -0.0793  -2.2366      results from matlab
    K1- old Human tracking trial 2 : 1.1573  1.2308 0.9893   2.4317 -0.0415  -2.1197      results from matlab (best result)
    K1- old Human tracking trial 2 : 1.1573  1.2308 0.9893  -0.7099  0.0415   2.1197      results after rotation from matlab        
    K1- new Human tracking trial 3 : 0.6967  0.6967 1.0146   2.3277 -0.0041  -2.1476      results from matlab -->
    
    <group ns="/head/kinect2">
        <param name="rgb_frame" value="/head/kinect2/rgb"/>
        <param name="depth_frame" value="/head/kinect2/depth"/>
        <param name="ir_frame" value="/head/kinect2/depth"/>-->
        <node name="startBody" pkg="k2_client" type="startBody" output ="screen"/>
    </group>
    
    <arg name="seg_sim" default="True"/>
    
    <node name="updatetf_with_kin" pkg="k2_client_vis" type="updatetf.py" respawn="false" output="screen">  
        <param name="seg_sim" value="$(arg seg_sim)"/>  
    </node>
    
  <node name="tf_to_hand" pkg="itia_kinova" type="tf_to_hand" output="screen" >
  </node>
  
  <arg name="hand" default="L"/>

  <node name="hum_goal_prediction_t2" pkg="itia_human_prediction" type="humpred.py" respawn="false" output="screen">
       <param name="user_hand" value="$(arg hand)"/>
  </node> 
  
  
</launch>
